io/driver/module_unload_load.py

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env python
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+#
+# See LICENSE for more details.
+#
+# Copyright: 2016 IBM
+# Author: Naresh Bannoth<nbannoth@in.ibm.com>
+#
+
+import time
+from avocado import main
+from avocado.utils import process
+from avocado.utils import linux_modules
+from avocado import Test
+
+
+class ModuleLoadUnload(Test):
+
+    """
+    PCI devices can be bound and unbound to drivers.
+    This test verifies that for given adapters.
+
+    :param module: Name of the driver module
+    :param iteration: Number of time to unload and load the module
+    """
+    def setUp(self):
+        """
+        get parameters.
+        """
+        self.module = self.params.get('module', default=None)
+        self.iteration = self.params.get('iteration', default=1)
+
+    def built_in_module(self):
+        """
+        checking whether the given module is built_in module or not
+        """
+        uname = process.getoutput("uname -r")
+        cmd = "cat /lib/modules/%s/modules.builtin" % uname
+        for each in str(process.getoutput(cmd)).split('\n'):
```

Comment:
use `genio.read_file()`

Reply:
Also, getting builtin modules can be added to `linux_modules` library.
Can be taken as TODO.